src/Aspire.Dashboard/Resources/Columns.Designer.cs

```diff
@@ -1,10 +1,10 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
```

Comment:
These should be in English. Not sure whey they are changed in this PR?

Reply:
Always because of the French

Reply:
done. I did not noticed my VS2022 preview was french ;). It would be great that default VS installer use English instead of local language.